book/graphics.md

```diff
@@ -534,7 +536,7 @@ redraw their window contents, though, to change what is displayed.
 [Chapter 13](animations.md) discusses compositing in more detail.
 
 
-But this scrolling is pretty slow.[^slow-scroll] Why? It turns out
+But scrolling in our browser is pretty slow.[^slow-scroll] Why? It turns out
```

Comment:
Otherwise it's ambiguous whether we're discussing scrolling in general or the specifics of scrolling in our browser. I think we are talking about our browser here.